src/app/table/table.component.ts

```diff
@@ -224,44 +221,44 @@ export class TableComponent implements OnInit {
     }
   }
 
-  getJobPercentFromPipeline(i, data){
+  getJobPercentFromPipeline(i, data) {
     try {
-        if (data[i].status == "success") {
-          return "95 5";
-        }
-        else if (data[i].status == "canceled") {
-          return "0 0";
-        }
-        else if (data[i].status == "pending") {
-          return "0 0";
-        }
-        else if (data[i].status == "failed") {
-          var count = 0;
-          for (var j = 0; j < data[i].jobs.length; j++) {
-            if(data[i].jobs[j].status == "success") {
-              count++;
-            }
+      if (data[i].status == "success") {
+        return "95 5";
```

Comment:
what we are returning here??

Reply:
This is returing the value to creating the graph